docs/advanced-topics/checksum-support.rst

```diff
@@ -29,25 +29,25 @@ Docker image repositories currently available, and what action on Dockstore is r
 
 |
 
-Once you perform the required action, you must also publish your entry in order to see the checksum info via the `TRS endpoints <https://dev.dockstore.net/api/static/swagger-ui/index.html#/GA4GHV20>`_.
+Once you perform the required action, you must also publish your entry in order to see the checksum info via the `TRS endpoints <https://dockstore.org/api/static/swagger-ui/index.html#/GA4GHV20>`_.
 Descriptions for the two endpoints of note are as follows:
 
-- To see all versions of an entry, use our `toolsGet <https://dev.dockstore.net/api/static/swagger-ui/index.html#/GA4GHV20/toolsGet>`_  endpoint and fill out the id parameter
-- To see a single version of an entry, go `here <https://dev.dockstore.net/api/static/swagger-ui/index.html#/GA4GHV20/toolsIdVersionsVersionIdGet>`_ and fill out id and version_id
+- To see all versions of an entry, use our `toolsGet <https://dockstore.org/api/static/swagger-ui/index.html#/GA4GHV20/toolsGet>`_  endpoint and fill out the id parameter
+- To see a single version of an entry, go `here <https://dockstore.org/api/static/swagger-ui/index.html#/GA4GHV20/toolsIdVersionsVersionIdGet>`_ and fill out id and version_id
 
 Just like the file endpoints, the id parameter used in the endpoints above can be found on an entry's public page; underneath the Info tab, look for the bolded words **TRS**.
 
 Tools
 -----
-As noted in the table above, Docker image checksums are grabbed on refresh and should work as long as the image is from Quay, Docker Hub,
+As noted in the table above, Docker image checksums are grabbed on refresh and should work as long as the image is from Quay.io, Docker Hub,
 or GitLab. It's also important to note that this is done for the Docker image registered on the tool through Dockstore and not necessarily
 the one included within the descriptor file itself.
 
 Workflows
 ---------
 For workflows, Docker image checksums are grabbed on snapshot, but this feature is not as reliable as the other checksum support covered so far. Although we can generally
```

Comment:
The sentence is a little blurry in that on first reading I can't tell if it's checksums or snapshots are not reliable.

Also, I don't know that would say "not reliable", which implies flakiness. It has limitations, but presumably if you code around the limitations, it is reliable.

Reply:
reworded